app/models/adopter.rb

```diff
@@ -156,7 +156,7 @@ def chimp_subscribe
       }
     end
 
-    if ((status == 'adopted') || (status == 'adptd sn pend'))
+    if adoptions.any?{ |a| a.relation_type_changed? && a.relation_type == 'adopted' } && dog_or_cat == "Dog"
       completed_date = Time.now.strftime('%m/%d/%Y')
       AdopterFollowupMailer.one_week_followup(id).deliver_later(wait: 7.days)
```

Comment:
The `AdopterFollowupMailer` probably not be triggered because the `status` of the `Adopter` and the `relation_type` of the `Adoption` each get set in the UI as a separate transaction.  So you'll need to asses the change of the relation_type directly when it is changed as its own separate action.   Bug filed as